x-pack/plugins/security_solution/common/endpoint/schema/trusted_apps.ts

```diff
@@ -35,7 +35,7 @@ export const GetTrustedAppsRequestSchema = {
 
 export const PostTrustedAppCreateRequestSchema = {
   body: schema.object({
-    name: schema.string({ minLength: 1 }),
+    name: schema.string({ minLength: 1, maxLength: 256 }),
```

Comment:
after some more thought, we could actually make this even shorter since it's just a name, however I don't think it will be likely that users actually create long trusted names.

I can also see from your screenshots that the table still truncates on a single line which is most important in regards to the look of the the feature, so I'm good with this as is.